File: scripts/reverse-engineering/find_decode.py

with open(r'c:\Users\Nicks\Desktop\Flyx-main\scripts\reverse-engineering\prorcp-unpacked.js', 'r', encoding='utf-8') as f:
    content = f.read()

print(f"Content length: {len(content)}")
index = content.find('decode')
while index != -1:
    print(f"Found 'decode' at {index}")
    # Print context
    start = max(0, index - 50)
    end = min(len(content), index + 50)
    print(f"Context: ...{content[start:end]}...")
    index = content.find('decode', index + 1)

File: scripts/reverse-engineering/find_decode.js

const fs = require('fs');
const content = fs.readFileSync('c:\\Users\\Nicks\\Desktop\\Flyx-main\\scripts\\reverse-engineering\\prorcp-unpacked.js', 'utf-8');

console.log(`Content length: ${content.length}`);
let index = content.indexOf('decode');
while (index !== -1) {
    console.log(`Found 'decode' at ${index}`);
    const start = Math.max(0, index - 50);
    const end = Math.min(content.length, index + 50);
    console.log(`Context: ...${content.substring(start, end)}...`);
    index = content.indexOf('decode', index + 1);
}
